githooks/pre-push.sh

```diff
@@ -48,7 +48,7 @@ check_bad_commit_msg() {
     range=$1
     pattern=$2
     msg=$3
-    bad_commit=$(git rev-list -n 1 --grep "$pattern" "$range")
+    bad_commit=$(git rev-list --grep "$pattern" "$range" | grep -v fbb3f6dad35497ed5dcea7e96d0a228b3e383741 | grep -v 4063bcdfe4fcd33b9dd43bf39bd8be3771274ced)
```

Comment:
```suggestion
    bad_commit=$(git rev-list --grep "$pattern" "$range" ^origin/master)
```
avoids needing to hard-code hashes and will automatically ignore anything that's already been pushed to master (I assume the problem is that, when creating a new branch, range starts at the zero hash and so includes everything; I think I even ran into that once).

Reply:
Thanks, that's a better solution. Yes, the problem is that for new branches this check is run for all existing commits.